src/settingsview.cpp

```diff
@@ -42,14 +42,12 @@ void SettingsView::init(int zoomPercent, const QString &downloadDir, const QStri
     ui->downloadDirPath->setText(downloadDir);
     ui->monitorDirPath->setText(monitorDir);
 }
-bool SettingsView::confirmDirDialog(const QString& dir, QString messageText, QString messageTitle)
+bool SettingsView::confirmDialog( QString messageText, QString messageTitle)
```

Comment:
It would be nice to have this refactoring (paired with the introduction of the `confirmDialogDownloadDir()` function) in a separate commit.